openedx/edx-ora2
# chore: [BB-6436] Make a small change on help text for instructor

JIRA: N/A
**What changed?**
Add a small change on help text of a ORA setting to make it clearer to instructor

**Developer Checklist**


**Testing Instructions**

- Checkout branch [open-craft:dat/BB-6436_push_instruction_to_ora_xblock](https://github.com/open-craft/edx-ora2/tree/dat/BB-6436_push_instruction_to_ora_xblock)
- Install edx-ora2 xblock on the feature branch to Studio (CMS)
- Restart CMS
- Add a ORA block in a unit
- Edit > Settings
- Make sure the changed text show up properly

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:


**PoW**

Install modified version of xblock in Studio and the text is updated as expected

![image](https://user-images.githubusercontent.com/8892458/177396916-1b95ee99-a916-48cb-b760-8fe9edb4e73e.png)

FYI: @openedx/content-aurora
